3cqsbot.py

```diff
@@ -248,10 +248,20 @@ def pair_data(account):
         logging.debug(error["msg"])
         sys.tracebacklimit = 0
         sys.exit("Problem fetching pair data from 3commas api - stopping!")
+
+    error, blacklist_data = p3cw.request(
+        entity="bots",
+        action="pairs_black_list"
+    )
+
+    if error:
+        logging.debug(error["msg"])
+        sys.tracebacklimit = 0
+        sys.exit("Problem fetching pairs blacklist data from 3commas api - stopping!")
     else:
```

Comment:
else is not necessary here, because it is stopped on error.

Reply:
Corrected